docs/migration/fastapi.rst

```diff
@@ -174,8 +174,273 @@ and to easily access dependencies from higher levels.
 
     * :doc:`/usage/dependency-injection`
 
+Lifespan
+~~~~~~~~
+
+If you're using an async context manager and pass parameters to it, most likely the order of parameters is inversed between FastAPI and Litestar.
+
+.. tab-set::
+
+    .. tab-item:: FastAPI
+        :sync: fastapi
+
+        .. code-block:: python
+
+            @asynccontextmanager
+            async def lifespan(
+                _app: FastAPI,
+                app_settings: AppSettings,
+            ):
+                # Setup code here
+                yield
+                # Teardown code here
+
+    .. tab-item:: Litestar
+        :sync: litestar
+
+        .. code-block:: python
+
+            @asynccontextmanager
+            async def lifespan(
+                app_settings: AppSettings,
+                _app: Litestar,
+            ):
+                # Setup code here
+                yield
+                # Teardown code here
+
+~~~~~~~
+Cookies
+~~~~~~~
```

Comment:
```suggestion

Cookies
~~~~~~~
```
is this meant to be a sub-section of lifespan

Reply:
nope it isnt, if I commit you're suggestion I'm fine ?

Reply:
Yeah I think that would be good to make it inline with all other sections